OracleDatabase/RAC/OracleRealApplicationClusters/dockerfiles/12.2.0.1/setupGrid.sh

```diff
@@ -26,12 +26,10 @@ chown -R grid:oinstall $GRID_HOME
 mkdir -p $INVENTORY
 chown -R grid:oinstall $INVENTORY
 
-############# Disabled sudo Access ######
-#### However, if user need the sudo access for his env, he can remove the comments from following lines and end rebuild the image ###############
-#chmod 666 /etc/sudoers
-#echo "oracle       ALL=(ALL)       NOPASSWD: ALL" >> /etc/sudoers
-#chmod 440 /etc/sudoers
-########## sudo code end here ###########
+chmod 666 /etc/sudoers
+echo "oracle       ALL=(ALL)       NOPASSWD: ALL" >> /etc/sudoers
```

Comment:
Do you really need to grant `oracle` full root access?

Reply:
Remove the Access for Oracle. However, I need access for Grid as default login user is grid.